src/python/pants/backend/python/rules/run_setup_py_test.py

```diff
@@ -1,7 +1,6 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
```

Comment:
There's little of interest in this file. In addition to changing to use `RuleRunner`, I did these things:

* Use `self.add_to_build_file()` instead of `self.create_file()` when it's writing a BUILD file. End behavior is the same, but better follows convention.
* Inlined two helper methods `assert_sources()` and `assert_owned()`.